tests/test_SklearnTfidfVectorizerConverterChar.py

```diff
@@ -0,0 +1,122 @@
+"""
+Tests scikit-learn's binarizer converter.
```

Comment:
tfidf not binarizer.

Reply:
Fixed. I think we should remove those comments. Errors due to copy paste and comments does not bring information.